apps/demo-app/src/views/DataGridView.vue

```diff
@@ -12,11 +23,40 @@ const data = [
   { id: 5, name: "Asperiks Kafelon", age: 99 },
 ];
 
-const features = computed(() => {
+const dummyFeature = createFeature(() => ({
+  name: Symbol("More actions"),
+  watch: [],
+  header: {
+    actions: () => [
+      {
+        iconComponent: h(OnyxSystemButton, {
+          label: "Column options",
+          icon: sort,
+          color: "medium",
+        }),
+        menuItems: [h(OnyxListItem, () => "Pin column"), h(OnyxListItem, () => "Unpin column")],
```

Comment:
The system button should use `OnyxMenuItem` since its used inside the flyout menu: https://storybook.onyx.schwarz/?path=/docs/buttons-systembutton-examples-flyoutmenu--docs